multiplayer/examples/outmatched/src/sounds/index.ts

```diff
@@ -0,0 +1,3 @@
+export const playSound = (_name: string) => {
+  // We don't have the license to freely redistribute sounds used in this game
```

Comment:
Would it make sense to keep the playSound code in the repo, but just fail silently and print to console if the sound file is not there?

It'd be great to have the exact same code in the example folder as used to upload Rune, both for transparency and for ease of uploading new versions of Outmatched.

Reply:
Tried that, but build fails because the imports doesn't exist. Could perhaps rewrite it in a way where they are (not) resolved at runtime instead of build time.

Reply:
Fixed in [1cf22d7](https://github.com/rune/rune-games-sdk/pull/149/commits/1cf22d72dd4e2adb4859a9686461487f54ef87b8)

Reply:
@oblador can you add the build step similar to the one in Sudoku?

Reply:
I would prefer if it worked the other way around, instead of this build having dependencies on private internal repos, we'd have a private internal repo that referenced this repo.

Reply:
Agreed to use a new repo for sounds, a separate deploy script in packages.json, a shared script file between games, temp checkout of repo via ssh link and fail for everyone

Reply:
I've migrated to use this repo now 👍